docs/spack/installing-spack.rst

```diff
@@ -3,7 +3,7 @@ Installing Spack
 ======================
 
 Follow the instructions from the `Spack Documentation 
-<http://https://spack.readthedocs.io/en/latest/getting_started.html/>`_.
+<http://spack.readthedocs.io/en/latest/getting_started.html/>`_.
```

Comment:
I think it still won't work because of the trailing `/`. And using https is nicer.

```suggestion
<https://spack.readthedocs.io/en/latest/getting_started.html>`_.
```